dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05380ChangeContainerPathToAbsoluteTest.java

```diff
@@ -0,0 +1,462 @@
+     * Should: Should not change anything
+     */
+    @Test
+    public void whenTemplateAdvancedHasRelativePath() throws IOException, DotDataException, DotSecurityException {
+        final String body = "#parseContainer(\"/application/containers/default\")";
+
+        final Host host = new SiteDataGen().nextPersisted();
+
+        final Contentlet theme = new ThemeDataGen().nextPersisted();
+        final Template template = new TemplateDataGen()
+                .title("template_test_" + System.currentTimeMillis())
+                .theme(theme)
+                .body(body)
+                .host(host)
+                .nextPersisted();
+
+
+        final Task05380ChangeContainerPathToAbsolute task05380ChangeContainerPathToAbsolute =
+                new Task05380ChangeContainerPathToAbsolute();
+
+        task05380ChangeContainerPathToAbsolute.executeUpgrade();
+
+        final ArrayList templates = getTemplateFromDataBase(template);
+
+        final HashMap templateMap = (HashMap) templates.get(0);
+        assertEquals(1, templates.size());
+
+        final String templateBody = templateMap.get("body").toString();
+        assertEquals(templateBody, body);
+    }
+
+    /**
+     * Method to Test: {@link Task05380ChangeContainerPathToAbsolute#executeUpgrade()}
+     * When: Exists A TemplateLayout with absolute path container from another site
+     * Should: Should not change anything
+     */
+    @Test
+    public void whenTemplateLayoutHasAbsolutePathToAnotherSite() throws IOException, DotDataException, DotSecurityException {
+
+        final Host host = new SiteDataGen().nextPersisted();
+        final Host anotherHost = new SiteDataGen().nextPersisted();
+
+        final String layout = String.format(jsonDrawBody, "//" + anotherHost.getHostname());
+        final String testBody = String.format(body, "//" + anotherHost.getHostname());
+
+        checkTemplateLayout(layout);
+        final Contentlet theme = new ThemeDataGen().nextPersisted();
+        final Template template = new TemplateDataGen()
+                .title("template_test_" + System.currentTimeMillis())
+                .theme(theme)
+                .drawedBody(layout)
+                .body(testBody)
+                .host(host)
+                .nextPersisted();
+
+
+        final Task05380ChangeContainerPathToAbsolute task05380ChangeContainerPathToAbsolute =
+                new Task05380ChangeContainerPathToAbsolute();
+
+        task05380ChangeContainerPathToAbsolute.executeUpgrade();
+
+        checkTemplateFromDataBase(anotherHost, template);
+    }
+
+    /**
+     * Method to Test: {@link Task05380ChangeContainerPathToAbsolute#executeUpgrade()}
+     * When: Exists A legacy TemplateLayout with absolute path container from another site
+     * Should: Should does not anything
+     */
+    @Test
+    public void whenLegacyTemplateLayoutHasAbsolutePathToAnotherSite() throws IOException, DotDataException, DotSecurityException {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6034685)